### PR TITLE
Introduce the developer hub

### PR DESCRIPTION
**Pull request by lukqw**
This PR overrides the default `sidebar-tree.html` and adds a separate nav section for our developer hub.

image

### CODE REVIEW

**Comment by lukqw** [comment]
I'd need help for two new icons here:
image

Taking a wild guess here and ping @thrau - for now I'm reusing some icons which are featured on the docs overview as well.

**Comment by lukqw** [comment]
@tinyg210 

Latest version should be enough to fulfill definition of completed for the bootstrapping project

image

File: content/en/developer-hub/_index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <div class="col-sm-12 col-md-6">
+      <a class="card card-tutorials" href="{{< ref "tutorials" >}}">
+        <div class="card-body">
+          <h5 class="card-title">Tutorials</h5>
+          <p class="card-text">Read through our tutorials to learn more about LocalStack.</p>
+        </div>
+      </a>
+    </div>
+    <div class="col-sm-12 col-md-6">
+      <a class="card card-applications" href="{{< ref "integrations" >}}">
```
Comment (alexrashed) [inline content/en/developer-hub/_index.html:41]: This link seems to be wrong, I think it should lead to "applications". 😉 
Just browsed through it a bit, lookin' good! 🚀

Comment (lukqw) [reply content/en/developer-hub/_index.html]: Thanks for the review :+1: 
Adjusted the link accordingly.

**Comment by steffyP** [comment]
there seem to be a broken link for the title-image of the last card:
Screenshot 2023-03-30 at 10 56 29

It's pointing to https://raw.githubusercontent.com/localstack/event-driven-architecture-with-amazon-sns-fifo/developer-hub/images/architecture.png - which seems to be wrong? 
@lukqw @HarshCasper

**Comment by lukqw** [comment]
Thanks a lot @steffyP, fixed accordingly!

File: data/developerhub/applications.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    {
+      "title": "Full-Stack application with AWS Lambda, DynamoDB & S3 for shipment validation",
+      "description": "Configure a CRUD web application for shipment validation & listing, and deploy it with Terraform on LocalStack",
+      "url": "https://github.com/localstack/shipment-list-demo",
+      "teaser": "https://raw.githubusercontent.com/localstack/shipment-list-demo/master/app_diagram.png",
+      "services": ["s3", "ddb", "lmb", "iam"],
+      "deployment":  ["terraform"],
+      "platform": ["java", "react"],
+      "tags": ["spring boot", "lambda-trigger"],
```
Comment (whummer) [inline data/developerhub/applications.json:22]: nit: I noticed that some of the tags use hyphens, whereas others don't. Should we maybe standardize on one common format?

**Review by whummer — APPROVED** [review]
Really awesome, love our new Developer Hub. 🤩  Great job on pushing this out, along with the initial set of polished samples! 🚀

**Pull request merged**